server/apps/immich/src/api-v1/user/dto/create-user.dto.ts

```diff
@@ -20,3 +20,24 @@ export class CreateUserDto {
   @ApiProperty({ example: 'Doe' })
   lastName!: string;
 }
+
+export class CreateAdminDto extends CreateUserDto {
+  @IsNotEmpty()
+  isAdmin!: true;
+}
+
+export class CreateUserOauthDto {
```

Comment:
Yes we need some kind of object to send between layers. Completely Agree about not needing the `ApiProperty` decorators. I did a copy paste since I wanted to keep the validation annotations since that would definitely be needed in the next step when we move validation to the domain
I'll get that updated

Reply:
Can you change this to have a capital `O` please - for consistency.

Reply:
Yes of course, sorry I forgot about this comment. I'll change that tomorrow 👍